dev/devicelab/lib/tasks/microbenchmarks.dart

```diff
@@ -95,5 +97,11 @@ Future<Map<String, double>> _readJsonResults(Process process) {
       jsonBuf.writeln(line.substring(prefixLength));
   });
 
+  process.exitCode.then((int code) {
```

Comment:
Should we `await` here? The call to `then` does not suspend execution, and so it's unclear who would catch the error thrown by the callback.